core/cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/jdbc/JdbcAuthenticationProperties.java

```diff
@@ -328,6 +357,22 @@ public void setSaltFieldName(final String saltFieldName) {
             this.saltFieldName = saltFieldName;
         }
 
+        public String getExpiredFieldName() {
```

Comment:
Also please update the docs in general to explain that password policy is possible now with JDBC to a limited extent.

And fix the build failures.